Scripts/Mend SCA/get-malicious-packages.sh

```diff
@@ -1,16 +1,27 @@
 #!/bin/bash
 # Prerequisites:
 # apt install jq curl
-# WS_USERKEY - An administrator's userkey
-# WS_EMAIL - The administrator's email
-# WS_APIKEY - API Key for organization
-# WS_API_URL - e.g. https://api-saas.mend.io/api/v2.0
+# MEND_USER_KEY - An administrator's userkey
+# MEND_EMAIL - The administrator's email
+# WS_APIKEY - API Key for organization (optional)
+# MEND_URL - e.g. https://api-saas.mend.io/api/v2.0
 
-# Log into API 2.0 and get the JWT Token
-JWT_TOKEN=$(curl -s -X POST --location "$WS_API_URL/login" --header 'Content-Type: application/json' --data-raw "{\"email\": \"$WS_EMAIL\", \"userKey\": \"$WS_USERKEY\", \"orgToken\": \"$WS_APIKEY\"}" | jq -r '.retVal.jwtToken' )
+# If API Key was not specified then exclude from Login request body.
+if [ -z "$WS_APIKEY" ]
+then
+        LOGIN_BODY="{\"email\": \"$MEND_EMAIL\", \"userKey\": \"$MEND_USER_KEY\" }"
```

Comment:
Add an echo here describing what happens such as the following "WS_APIKEY variable not found, login api will default to the last organization accessed via the webui if the userkey has access to mulitple organizations"